tests/e2e/mnist.py

```diff
@@ -12,22 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# In[]
 import os
 
 import torch
 from pytorch_lightning import LightningModule, Trainer
 from pytorch_lightning.callbacks.progress import TQDMProgressBar
+from pytorch_lightning.loggers import CSVLogger
```

Comment:
Is this import used for anything?

Reply:
I guess no, can be removed :+1: 
Thanks @sutaakar for reviewing!